### ISSUE TITLE
gw-workers utility is broken since we moved secrets from tc-secrets to password store

### ISSUE THREAD

**Issue by petemoore** [taskcluster/taskcluster#3518]
**Describe the bug**
The gw-workers tool no longer works.

We should:

* move it to tools directory
* rename it to `workers` instead of `gw-workers` (in anticipation of having it working for all worker implementations)
* not call AWS/GCP directly, but interface with worker manager to get workers
* use secrets from local password store, rather than from taskcluster secrets service

**Comment by lotas** [comment]
@petemoore is it still a thing? I don't see gw-workers in the code anymore

**Comment by petemoore** [comment]
Oh, interesting, I'd forgotten about this tool! I wonder where it has gone, and if it was removed at some point. I wouldn't mind keeping it alive, or moving it e.g. to the taskcluster cli, as it was a super way of interfacing with workers.

**Comment by lotas** [comment]
True, it moved to separate repo - https://github.com/taskcluster/community-tc-utils
